src/select/parts/plain-list.tsx

```diff
@@ -40,10 +44,19 @@ const PlainList = (
     listBottom,
     useInteractiveGroups,
     screenReaderContent,
+    firstOptionSticky,
   }: SelectListProps,
   ref: React.Ref<SelectListProps.SelectListRef>
 ) => {
+  const [width, menuMeasureRef] = useContainerQuery(
```

Comment:
In VirtualList there was already a container query that I could reuse for this, but here in PlainList I had to add it.